sk/lessons/basics/enum.md

```diff
@@ -49,11 +49,11 @@ Táto funckia má niekoľko možností použitia, pozrite sa do oficiálnej doku
 
 ### chunk_by
 
-Ak potrebujete rozdeliť kolekciu do menších na základe niečoho iného, než ich veľkosť, použite funkciu `chunk_by`:
+Ak potrebujete rozdeliť kolekciu do menších na základe niečoho iného, než ich veľkosť, použite funkciu `chunk_by`. Ako argumenty funkcia berie kolekciu (Enum) a funkciu - ak sa zmení jej výstup, ukončí sa aktuálna podkolekcia a začne sa vytvárať ďalšia:
 
 ```elixir
-iex> Enum.chunk_by(["one", "two", "three", "four", "five"], fn(x) -> String.length(x) end)
```

Comment:
I agree but it would be hard to go back and remove all the original first examples from the other translations and eventually they may end up mis-synced. But if the people responsible for that piece that have already done the changes to catch up dont mind changing it, then its no problem. I'm not sure how many have started on this small bit. From the chart it looks like only a few but I'm thinking in broad safe-rather-than-sorry terms.